css-three-column-layout/styles.css

```diff
@@ -0,0 +1,70 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-basis: auto;
```

Comment:
These can be removed, they don't do anything, since `container` is not a `flexbox` element.